tst.txt

```diff
@@ -1,4 +1,6 @@
 Work my lovely
-I beg you pls
```

Comment:
Unrelated change. Should be in another PR.

Reply:
I addressed the issue!

Reply:
I took a look. Looks good!